ReleaseChecklist.md

```diff
@@ -1,24 +1,52 @@
-Checklist for making a release:
+## Checklist for making a release:
 
+### Requirements
+ - [ ] AppVeyor account
```

Comment:
You can also just login through github. Is it actually needed?

Reply:
That's true. But you'd need access to the project. But I think everyone has that. I can remove this requirement.

Reply:
I mean where do you actually need to access appveyor?